paasta_tools/marathon_tools.py

```diff
@@ -185,8 +188,18 @@ def __repr__(self):
     def copy(self):
         return self.__class__(self.service, self.instance, dict(self.config_dict), dict(self.branch_dict))
 
+    def close_zookeeper_client(self):
+        self.zookeeper_client.stop()
+        self.zookeeper_client.close()
+
+    def get_min_instances(self):
+        return self.config_dict.get('instances', 1)
```

Comment:
I'm concerned about potential confusion here. What if this was literally `min_instances` in the config, to make it very obvious which things are autoscaling and which are static?